test/libsolidity/syntaxTests/immutable/conditionally_initialized_multiple_writes.sol

```diff
@@ -0,0 +1,14 @@
+contract C {
+    uint immutable multipleWrites;
+
+    constructor() {
+        if (true) {
+            multipleWrites = 1;
+        } else {
+            multipleWrites = 1;
+        }
+        multipleWrites = 2; // error
```

Comment:
Is there a "happy case" for this, i.e. a test like this one here, except without the third write, e.g.

```
contract C {
    uint immutable multipleWrites;

    constructor() {
        if (true) {
            multipleWrites = 1;
        } else {
            multipleWrites = 2;
        }
    }
}
```

Reply:
Good call. Updated all test cases to add such cases.